### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by nick-Ag**
Update readme.

## Testing

Testing not required as it is just a README change.

### CODE REVIEW

**Comment by nick-Ag** [comment]
A small start to redoing our readme, I added a big Segment logo to the top, made the description of what destination actions is larger, and linked out to some of our existing repo docs from the getting started page

**Review by mericsson — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,8 +1,16 @@
+<img src="https://brand.segment.com/site-assets/03d0de2f/images/brand-guidelines/content/twilio/twilio-segment-color-logo-white-2x.png" style="width:100%; height=100%" alt="Segment logo" >
```
Comment (mericsson) [inline README.md:1]:
```suggestion
<img src="https://brand.segment.com/site-assets/03d0de2f/images/brand-guidelines/content/twilio/twilio-segment-color-logo-white-2x.png" style="width:50%; height=50%" alt="Twilio Segment logo" >
```

File: README.md [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
+
 # Action Destinations
 
-Action Destinations are a way to build streaming destinations on Segment. To begin, follow the instructions in Get Started below.
+Action Destinations are a way to build streaming destinations on Segment.
```
Comment (mericsson) [inline README.md:5]:
```suggestion
Action Destinations are the new way to build streaming destinations on Segment.
```

File: README.md [hunk 3]
```diff
@@ -1,8 +1,16 @@
+<img src="https://brand.segment.com/site-assets/03d0de2f/images/brand-guidelines/content/twilio/twilio-segment-color-logo-white-2x.png" style="width:50%; height=50%" alt="Twilio Segment logo" >
+
 # Action Destinations
 
-Action Destinations are a way to build streaming destinations on Segment. To begin, follow the instructions in Get Started below.
+Action Destinations are the new way to build streaming destinations on Segment.
+
+Action Destinations allows builders to easily create Segment integrations with partner tools (e.g. Salesforce) by leveraging a first-class CLI to reduce boilerplate, a strong type-safety framework to enable the IDE and comiler to provide guidance and auto-complete.
```
Comment (mericsson) [inline README.md:7]:
```suggestion
Action Destinations were [launched in December 2021](https://segment.com/blog/introducing-destination-actions/) to enable customers with a customizable framework to map Segment events to their favorite 3rd party tools. This repository contains the Action Destination Definitions as well as a CLI to generate the scaffolding for new destinations and run unit tests. If you'd like to contribute, please review the [Contributing Guide](./CONTRIBUTING.md).
```

File: README.md [hunk 4]
```diff
@@ -1,8 +1,16 @@
...
+
+Integrations built with Action Destinations allow users to transparently view the data they are sending to the downstream destination. Users can easily customize how data is delivered via action mappings and decide which events trigger their actions with an intuitive subscription system.
```
Comment (mericsson) [inline README.md:9]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -1,8 +1,14 @@
+<img src="https://brand.segment.com/site-assets/03d0de2f/images/brand-guidelines/content/twilio/twilio-segment-color-logo-white-2x.png" style="width:50%; height=50%" alt="Twilio Segment logo" >
+
 # Action Destinations
 
-Action Destinations are a way to build streaming destinations on Segment. To begin, follow the instructions in Get Started below.
+Action Destinations are the new way to build streaming destinations on Segment.
+
+Action Destinations were [launched in December 2021](https://segment.com/blog/introducing-destination-actions/) to enable customers with a customizable framework to map Segment events to their favorite 3rd party tools. This repository contains the Action Destination Definitions as well as a CLI to generate the scaffolding for new destinations and run unit tests. If you'd like to contribute, please review the [Contributing Guide](./CONTRIBUTING.md).
```
Comment (mericsson) [inline README.md:7]:
```suggestion
Action Destinations were [launched in December 2021](https://segment.com/blog/introducing-destination-actions/) to enable customers with a customizable framework to map Segment event sources to their favorite 3rd party tools.

This repository contains the Action Destination Definitions as well as a CLI to generate the scaffolding for new destinations and run unit tests. If you'd like to contribute, please review the [Contributing Guide](./CONTRIBUTING.md).
```

**Comment by mericsson** [comment]
Made some tweaks. I think ready to merge now.

File: README.md [hunk 6]
```diff
@@ -11,7 +19,7 @@ Fore more detailed instruction, see the following READMEs:
 - [Mapping Kit](./packages/core/src/mapping-kit/README.md)
 - [Destination Kit](./packages/core/src/destination-kit/README.md)
 
-Table of Contents:
+# Table of Contents:
```
Comment (jinapark202) [inline README.md:22]:
```suggestion
## Table of Contents:
```

File: README.md [hunk 7]
```diff
@@ -83,6 +91,8 @@ In order to run the CLI (`./bin/run`), your current working directory needs to b
 
 For specific information about each CLI command, please refer to this [README](https://github.com/segmentio/action-destinations/tree/main/packages/cli).
 
+For instructions on how to create a new integration, see the [Create a new Destination Action](./docs/create.md) docs
```
Comment (jinapark202) [inline README.md:94]:
```suggestion
For instructions on how to create a new integration, see the [Create a new Destination Action](./docs/create.md) docs.
```

**Review by jinapark202 — APPROVED** [review]
couple minor suggestions. Also, can we get rid of the canary testing section in the testing doc? I don't think it's relevant anymore

**Pull request merged**